GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -124,6 +124,30 @@ public override bool TryGetGVFSEnlistmentRoot(string directory, out string enlis
             return POSIXPlatform.TryGetGVFSEnlistmentRootImplementation(directory, out enlistmentRoot, out errorMessage);
         }
 
+        public override bool TryGetDefaultLocalCacheRoot(string enlistmentRoot, out string localCacheRoot, out string localCacheRootError)
+        {
+            string homeDirectory = Environment.GetEnvironmentVariable("HOME");

```

Comment:
This *could* throw a `SecurityException` if user doesn't have permissions to perform the operation.

Reply:
Good point, I will catch that and set an appropriate error message.